app/controllers/api/base_controller.rb

```diff
@@ -36,6 +36,11 @@ class BaseController < ApplicationController
                          :only => [:show, :update, :destroy, :handle_options_request, :options]
     end
     skip_before_action :get_global_session_data
+    skip_before_action :reset_toolbar
+    skip_before_action :set_session_tenant
+    skip_before_action :set_user_time_zone
+    skip_before_action :set_gettext_locale
```

Comment:
@abellotti can we go over that method in more detail? [here](https://github.com/ManageIQ/manageiq/blob/master/app/controllers/application_controller.rb#L2413-L2434) it is for reference. This method references `current_user` (not `User.current_user`) which will always be `nil` for an API request. So paths that reference the user's settings will not get executed.... The remaining stuff has to do with either getting the locale from the headers or from the `MiqServer` configuration. 

Perhaps I can write some tests for this since skipping this callback didn't break any? Has internationalization only started to be implemented in /api ?